modules/influxdb/src/test/java/org/testcontainers/containers/InfluxDBContainerV1WithUserTest.java

```diff
@@ -1,44 +1,43 @@
 package org.testcontainers.containers;
 
+import java.util.concurrent.TimeUnit;
 import org.influxdb.InfluxDB;
 import org.influxdb.dto.Point;
 import org.influxdb.dto.Query;
 import org.influxdb.dto.QueryResult;
 import org.junit.Rule;
 import org.junit.Test;
 
-import java.util.concurrent.TimeUnit;
```

Comment:
Could you please revert this change? `java.*` imports should go after non-JDK ones, not before